src/hardware/input/mouse_config.cpp

```diff
@@ -153,6 +177,52 @@ static void SetComModel(const std::string_view model_str)
 	assert(result);
 }
 
+static void SetDosDriverOptions(const std::string_view options_str)
```

Comment:
Static function names are lower_snake_case.

Reply:
Ok, changed all the names in the file.